tsconfig.json

```
@@ -10,6 +10,14 @@
 		"resolveJsonModule": true,
 		"target": "esnext",
 
+		/* Project Imports */
+		"baseUrl": ".",
+		"paths": {
+			"@wordpress/*": [
+				"packages/*/src/index.js"
```

Would it work as follows?
```suggestion
				"packages/*/index.js",
				"packages/*/src/index.js"
```

---

I think that or the inverse order should work. If there are specific packages that don't adhere to one of these, they could be listed independently as literal `"@wordpress/package-name": [ "packages/package-name/path/to/main" ]`.